docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +162,235 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+* Notifications for child-devices and their external devices are handled differently, as configuration files are fetched/pushed from/to external devices via HTTP filetransfer. Thereby notifications are part of the filetransfer contract (details see section [Fetching/Pushing configuration file](#fetchingpushing-configuration-file-fromto-the-external-device) below).
+
+## Configuration files for child devices
+
+To manage configuration files for child-devices the `c8y_configuration_plugin` aspects as below are relevant:
+
+* Associating with cloud's child-device twin, i.E. managing supported configuration list of child devices 
```

Comment:
```suggestion
* Associating with cloud's child-device twin, i.e. managing supported configuration list of child devices 
```

Reply:
That section way anyway entirely rephrased.